### ISSUE TITLE
Cache posthog events on S3 or some other persistent storage instead of caching them on disk

### ISSUE THREAD

**Issue by Martinsos** [wasp-lang/wasp-bot#20]
Right now we cache fetched posthog events on disk, in a .json file.
Problem is, in production, when deployed to Heroku, this file is lost whenever a dyno (Heroku instance) is restarted, which can happen quite often on Heroku, possibly even daily (Heroku restarts dynos often to keep them "healthy").

Solution is to store the file somewhere on persistent storage -> most likely on S3.

**Comment by FranjoMindek** [comment]
We are on fly now. Do we still want to do this issue? @Martinsos

**Comment by Martinsos** [comment]
Yeah @infomiho solved this by putting us on Fly -> I think we are good for now, no need to push for this at the moment. Would be cool still, so we don't have to refetch all of them if server gets reset/destroyed, but not a priority I think. I would leave the issue open for now I think. We can Icebox it though.
